sysutils/edid-decode/Portfile

```
@@ -0,0 +1,17 @@
+PortSystem       1.0
+
+name             edid-decode
+version          git-e48fb38
+categories       sysutils
+platforms        darwin
+maintainers      @wwalexander
```

This should also be marked `openmaintainer` unless there is a reason why specifically only you should be allowed to modify this port.  Additionally, your Github tag and optional email address (if included) should be surrounded by curly braces.

```suggestion
maintainers      {@wwalexander} openmaintainer
```

---

Only multi-value entries (such as those having both a GitHub handle and an obfuscated email address) should be surrounded by curly braces.

---

I fixed this, omitting the curly braces (d73a890).